components/access/RegistrationDialog.vue

```diff
@@ -1,144 +1,146 @@
 <template>
-  <div class="registration">
-    <v-container class="registration__container rounded">
-      <v-row justify="space-between ml-0 mr-0">
-        <h2>
-          <!--{{ $t('access.registration.title') }}-->
-          Регистрация
-        </h2>
-        <v-btn text class="mt-1 text-capitalize buttonTo" color="#00AC00" @click="toSignInDialog">
-          <span>
-            <!--{{ $t('access.login.linkAndButtonName') }}-->
-            Вход
-          </span>
-        </v-btn>
-      </v-row>
-      <p class="pt-4 description">
-        <!--{{ $t('access.registration.description') }}-->
-        Зарегистрируйтесь через социальные сети
-        или электронную почту
-      </p>
-      <div class="d-flex justify-space-between">
-        <v-btn
-          width="31%"
-          color="rgba(0, 0, 0, 0.2)"
-          outlined
-          rounded
-        >
-          <img src="@/assets/images/social-icons/vk.svg">
-        </v-btn>
-        <v-btn
-          width="31%"
-          color="rgba(0, 0, 0, 0.2)"
-          outlined
-          rounded
-        >
-          <img src="@/assets/images/social-icons/instagram.svg">
-        </v-btn>
-        <v-btn
-          width="31%"
-          color="rgba(0, 0, 0, 0.2)"
-          outlined
-          rounded
-        >
-          <img src="@/assets/images/social-icons/google.svg">
-        </v-btn>
-      </div>
-      <!--      :placeholder="$t('access.global.emailPlaceholder')"-->
-      <v-text-field
-        class="mt-5 textField"
-        dense
-        rounded
+  <v-container class="registration__container rounded">
+    <v-row justify="space-between ml-0 mr-0">
```

Comment:
you mixed up between the props
```suggestion
    <v-row justify="space-between" class="mx-0">
```